src/main/java/com/guysfromusa/carsgame/v1/CarResource.java

```diff
@@ -82,7 +83,8 @@ public Car addCar(@PathVariable("name") String name, @RequestBody CarType carTyp
             @ApiResponse(code = 400, message = "Car not found"),
             @ApiResponse(code = 400, message = CAR_CRASHED_MESSAGE),
             @ApiResponse(code = 400, message = CAR_EXISTS_IN_GAME_MESSAGE),
```

Comment:
can we use (HttpStatus.)SC_BAD_REQUEST instead of 400 ?

Reply:
I'll check